lib/BackgroundJobs/ClassifierJob.php

```diff
@@ -29,6 +30,8 @@ public function __construct(ITimeFactory $time, LoggerInterface $logger, QueueSe
 		$this->userMountCache = $userMountCache;
 		$this->jobList = $jobList;
 		$this->config = $config;
+		$this->setInterval(60 * 15);
+		$this->setTimeSensitivity(self::TIME_INSENSITIVE);
```

Comment:
Might be nice to make this configurable somehow, so that if there are enough resources available you can still run them also during usual operation times.

Reply:
From this description it seems optional: https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/background_jobs_configuration.html